### PR TITLE
Fix creation of log files

### PR DESCRIPTION
**Pull request by pablomarga**
Hi team this PR solves a problem related with the creation of the log files, adding a condition that if the file does not exist it will be created. Also solves a problem with the rotation of the log file if it's size is over 100MB.

### CODE REVIEW

**Review by gabiwassan — APPROVED** [review]
LGTM 👍🏼 
CD ✅ 
Testting ✅

File: server/lib/base-logger.ts [hunk 1]
```diff
@@ -33,11 +33,15 @@ export class BaseLogger {
...
+  PLAIN_LOGS_FILE_NAME: string = '';
   RAW_LOGS_PATH: string = '';
+  RAW_LOGS_FILE_NAME: string = '';
 
-  constructor(plainLogsPath: string, rawLogsPath: string) {
-    this.PLAIN_LOGS_PATH = plainLogsPath;
-    this.RAW_LOGS_PATH = rawLogsPath;
+  constructor(plainLogsFile: string, rawLogsFile: string) {
+    this.PLAIN_LOGS_PATH = path.join(WAZUH_DATA_LOGS_DIRECTORY_PATH, plainLogsFile);
+    this.RAW_LOGS_PATH = path.join(WAZUH_DATA_LOGS_DIRECTORY_PATH, rawLogsFile);
```
Comment (Desvelao) [inline server/lib/base-logger.ts:42]: Could we move the variable that stores the path of these files to the `constants.ts` file?

File: server/lib/filesystem.ts [hunk 2]
```diff
@@ -8,6 +8,20 @@ export const createDirectoryIfNotExists = (directory: string): void  => {
...
+    fs.writeFileSync(
+      filePath,
+      JSON.stringify({
+        date: new Date(),
+        level: 'info',
+        location: 'logger',
+        message: 'Log file creation',
+      }) + '\n'
+    );
+  };
```
Comment (Desvelao) [inline server/lib/filesystem.ts:22]: **question**: Why do we need to add a log to the file when it has been created?

**Review by eze9252 — APPROVED** [review]
LGTM!

**Comment by Desvelao** [comment]
**issue**: The filename looks a little bit weird.
image

**suggestion**: Here it is the filename had before the changes: https://github.com/wazuh/wazuh-kibana-app/blob/12ad00eedf03e6625493b05d8ba9a9e6fff48a4e/server/lib/logger.ts#L105-L108

```<FILENAME>.<DATE_IN_TIMESTAMP>.<EXTENSION>```

**Review by Desvelao — APPROVED** [review]
**review**
Code review: ✅ 
Testing: ✅

**Pull request merged**